### PR TITLE
Support providing hasMany relationships on incoming data

### PR DESCRIPTION
**Pull request by Marlinc**

### CODE REVIEW

**Comment by bravo-kernel** [comment]
Failing Travis build is unrelated and already fixed in #13.

Based on lessons learned it would nice if a multi-word attribute could be added to the test. So far I have used incoming `dummy-counter` which should convert to `dummy_counter`.

**Comment by Marlinc** [comment]
Sorry for the missing description, was a bit quick in creating the PR. Currently there is no conversion in place that takes a incoming field called `dummy-counter` and turns it into `dummy_counter`.

File: src/Listener/JsonApiListener.php [hunk 1]
```diff
@@ -731,6 +731,22 @@ protected function _convertJsonApiDocumentArray($document)
         foreach ($document['data']['relationships'] as $key => $details) {
             // skip hasMany relationships for now
```
Comment (ceeram) [inline src/Listener/JsonApiListener.php]: can drop the comment now :)

Comment (Marlinc) [reply src/Listener/JsonApiListener.php]: You're right! ;)

**Comment by bravo-kernel** [comment]
@Marlinc it would be nice if that could be added since I have already started implementing it here: https://github.com/bravo-kernel/crud-json-api/blob/b7dc044f49719b8c07e7a9edd2c54b5bd78cd55e/src/Listener/JsonApiListener.php#L726

File: src/Listener/JsonApiListener.php (hunk 1 above)
Comment (bravo-kernel) [reply src/Listener/JsonApiListener.php]: indeed ✅

**Comment by ceeram** [comment]
yay

**Comment by Marlinc** [comment]
I've rebased on master ;)

**Comment by bravo-kernel** [comment]
@Marlinc thanks for adding the inflection, much appreciated 💃

**Comment by bravo-kernel** [comment]
I will merge and create a new release 0.2.0 once the green box pops up.

Much appreciated 💯

**Comment by Marlinc** [comment]
:+1:

**Comment by bravo-kernel** [comment]
https://github.com/FriendsOfCake/crud-json-api/releases/tag/0.2.0

**Pull request merged**